pkg/controllers/deprovisioning/emptiness.go

```diff
@@ -65,8 +69,16 @@ func (e *Emptiness) ShouldDeprovision(ctx context.Context, n *state.Node, provis
 }
 
 // ComputeCommand generates a deprovisioning command given deprovisionable nodes
-func (e *Emptiness) ComputeCommand(_ context.Context, nodes ...CandidateNode) (Command, error) {
-	emptyNodes := lo.Filter(nodes, func(n CandidateNode, _ int) bool { return len(n.pods) == 0 })
+func (e *Emptiness) ComputeCommand(ctx context.Context, nodes ...CandidateNode) (Command, error) {
```

Comment:
I don't think we need to respect PDBs here because the node should be empty. Is this right?

Reply:
Good catch